docs/prepping-local-system-runbook-linux.md

```diff
@@ -228,176 +217,167 @@ This gcloud configuration is called [default]. You can create additional configu
+    `$ sudo apt-get install docker-ce docker-ce-cli containerd.io`
 
-enabled: **false**
-```
+3. To create a `docker` group of users, enter:
 
-5. Save the file.
+    `$ sudo groupadd docker`
 
-Add any known, customized settings for Pega to your deployment
---------------------------------------------------------------
+4. To add your user name to the docker group, enter:
 
-The Pega deployment model supports advanced configurations to fit most existing clients' needs. If you are a Pega client and have known, required customizations for your deployment and you already use the following files to add your known customizations, you can copy those configurations into the configuration files Pega added for this purpose in the pega-helm-charts repository folder, \<local filepath\>/\<platform\>-demo/pega-helm-charts-master/charts/pega/config/deploy:
+    `$ sudo usermod -aG docker $USER`
 
-- context.xml: add additional required data sources
+5. Log out and log back in so that your group membership is re-evaluated.
 
-- prlog4j2.xml: modify your logging configuration, if required
+6. To verify that the dpocker command runs without the `sudo` preface and that Docker CE is installed correctly by running the hello-world image, enter:
 
-- prconfig.xml: adjust the standard Pega Platform configuration with known,
-    required settings
+    `$ docker run hello-world`
 
-Make these changes before you begin deploying Pega Platform using Helm charts.
+    These installation instructions are sourced from the Docker documentation,  [Install Docker Engine on Ubuntu](https://docs.docker.com/v17.09/engine/installation/linux/docker-ce/ubuntu/). For additional information and links to other Linux distributions, see the instructions provided by other supported Linux distributions.
 
 Creating SSL certificates for HTTPS load balancer on a Kubernetes ingress
 -----------------------------------
 
-When you configure an HTTPS load balancer through a Kubernetes ingress, you can configure the load balancer to present authentication certificates to the client. After you generate your certificates and corresponding private keys using any SSL Certificate creation tool, you pass the certificates and private keys to your Kubernetes ingress using the method supported for the specific environment. For Pega support details, see the [Addons Helm chart](https://github.com/pegasystems/pega-helm-charts/tree/master/charts/addons). 
+When you configure an HTTPS load balancer through a Kubernetes ingress, you can configure the load balancer to present authentication certificates to the client. After you generate your certificates and corresponding private keys using any SSL Certificate creation tool, you pass the certificates and private keys to your Kubernetes ingress using the method supported for the specific environment. For Pega support details, see the [Addons Helm chart](https://github.com/pegasystems/pega-helm-charts/tree/master/charts/addons).
```

Comment:
Is this required for all installations?

Reply:
it is not...we state "you can configure the load balancer". We have to document this for clients in case a client wants to use HTTPS certificates.